src/librustdoc/clean/types.rs

```diff
@@ -1902,6 +1923,15 @@ crate enum Variant {
     Struct(VariantStruct),
 }
 
+impl Variant {
+    crate fn has_stripped_fields(&self) -> Option<bool> {
+        match *self {
+            Self::Struct(ref _struct) => Some(_struct.has_stripped_fields()),
+            Self::CLike | Self::Tuple(_) => None,
```

Comment:
I think tuple variants can have stripped fields too.

Reply:
Also, why does this function need to return an Option?

Reply:
> I think tuple variants can have stripped fields too.

This is a good point. I wonder why it wasn't handled before...

> Also, why does this function need to return an Option?

To differentiate between items that can have stripped fields and the rest.

Reply:
But why do we need to differentiate?

Reply:
Yes because of json.